instrumentserver/apps.py

```diff
@@ -58,10 +59,17 @@ def serverScript() -> None:
     # Load and process the config file if any.
     configPath = args.config
 
-    stationConfig, serverConfig, guiConfig, tempFile = None, None, None, None
+    stationConfig, serverConfig, guiConfig, tempFile, pollingRates = None, None, None, None, None
     if configPath != '':
         # Separates the corresponding settings into the 4 necessary parts
-        stationConfig, serverConfig, guiConfig, tempFile = loadConfig(configPath)
+        stationConfig, serverConfig, guiConfig, tempFile, pollingRates = loadConfig(configPath)
+
+    serverScript.pollingThread = QtCore.QThread()
```

Comment:
We should probably have this only start if the user passes a polling rate. But we can have that as an issue for now if its too hard